backend/beets_flask/server/routes/library/resources.py

```diff
@@ -743,7 +743,13 @@ def _rep_Album(
 
     out: dict[str, Any] = dict()
 
-    out["path"] = beets_util.displayable_path(album.path)
+    try:
+        out["path"] = beets_util.displayable_path(album.path)
```

Comment:
We have to think about if we want to include the path anyways. As this always triggers a item lookup i.e. db join and is quite inefficient.

At least for minimal returns there is an argument in omitting the path.